PerDiem/PerDiem/Controllers/TransactionsViewController.m

```diff
@@ -204,10 +224,14 @@ - (NSArray *)rightButtons
 
 - (void)swipeableTableViewCell:(SWTableViewCell *)cell didTriggerRightUtilityButtonWithIndex:(NSInteger)index {
     NSIndexPath *indexPath = [self.tableView indexPathForCell:cell];
+    NSInteger indexPathNumber = indexPath.row;
+    if (self.budget || self.period) {
```

Comment:
We don't want to show the loader in the transactions list when we're in the calendar/day view transition. `self.period` indicates that we're in a specific timeframe (day), right?